Userland/Libraries/LibWeb/Painting/PaintableBox.cpp

```diff
@@ -279,8 +279,9 @@ Optional<PaintableBox::ScrollbarData> PaintableBox::compute_scrollbar_data(Scrol
     auto scrollport_size = direction == ScrollDirection::Horizontal ? padding_rect.width() : padding_rect.height();
     if (scroll_overflow_size == 0)
         return {};
-
-    auto thumb_length = scrollport_size * (scrollport_size / scroll_overflow_size);
+    auto min_thumb_length = 50;
+    //Insure thumb length is at least 50px.
```

Comment:
Typo: "Insure" -> "Ensure"

Reply:
Thanks Sergey!  I did fix the typo but I didn't notice your PS.  I can remove the comment too since as you say it's a little redundant.